ods_ci/tests/Resources/Page/ODH/ODHDashboard/ODHDashboardAPI.resource

```diff
@@ -70,6 +70,8 @@ Perform Dashboard API Endpoint POST Call
     &{args}=       Create Dictionary     url=${ODH_DASHBOARD_URL}/${endpoint}   expected_status=any
     ...             headers=${headers}   json=${payload}    timeout=5  verify=${False}
     ${response}=    Run Keyword And Continue On Failure     Perform Request     POST     &{args}
+    Log    msg= &{args}  level=DEBUG
+    Log    msg= ${response}  level=DEBUG
```

Comment:
could u pls remove these lines after you finish testing the PR?

Reply:
Done